assets/armsnippets.jsonc

```diff
@@ -120,6 +120,35 @@
         ],
         "description": "Nested Deployment"
     },
+    "Nested Deployment": {
+        "prefix": "arm-nested-template",
+        "body": [
+            "{",
+            "\t\"name\": \"${1:nestedDeployment1}\",",
+            "\t\"type\": \"Microsoft.Resources/deployments\",",
+            "\t\"apiVersion\": \"2019-10-01\",",
+            "\t\"properties\": {",
+            "\t\t\"expressionEvaluationOptions\": {",
+            "\t\t\t\"scope\": \"${2|inner,outer|}\"",

```

Comment:
Do you think the default should be inner or outer?

Reply:
Good point. Looking at docs, outer is the default. I will re-order.

Unfortunately, I've inadvertently toasted the local branch and will need to open a new PR.